Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -210,6 +244,24 @@ bool IsPointOutsideCalibration(Common::DVec2 point, ControllerEmu::ReshapableInp
   return current_radius > input_radius * ALLOWED_ERROR;
 }
 
+void DrawVirtualNotches(QPainter& p, ControllerEmu::ReshapableInput& stick, QColor gate_brush_color)
+{
+  const double segment_size = stick.GetVirtualNotchSize();
+  if (segment_size <= 0.0)
+    return;
+
+  p.setBrush(gate_brush_color.darker(125));
+  for (int i = 1; i <= 8; ++i)
```

Comment:
Looping from 1 to 8 is kinda unusual. I'd change this to start at 0 and stop at 8.

Reply:
I originally implemented it to start at 0 but it doesn't render properly (more than likely due to the `lower_angle` being negative) - I can either add Tau to all the angles and start at 0, or start from 1 (and perhaps leave a comment there for clarity?)

Reply:
Ah. Either adding tau or a comment works. Maybe I'll fix the `Radius` getters with negative angles at some point.

Reply:
Added some code to change the angle from within the segment logic and added a comment to explain as well:
```c++
  // We subtract from the provided direction angle so it's better
  // to add Tau here to prevent a negative value instead of
  // expecting the function call to be aware of this internal logic
  const double center_angle = direction + MathUtil::TAU;
```

It may be a bit verbose but at least it gets the point across I guess 🤷‍♂️